lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential.py

```diff
@@ -178,8 +210,7 @@ def main():
         user=dict(),
         team=dict(),
         kind=dict(required=True,
-                  choices=["ssh", "net", "scm", "aws", "rax", "vmware", "satellite6",
```

Comment:
A few things of note that got fixed w/ the compat shim above:
- Vault is a top-level kind now, but this _still_ allows you to use the old style `{kind: ssh, vault_password: xyz}`
- Rax + Azure support is gone in modern Towers.